plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -171,6 +171,13 @@ public void testAddColumn()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    @Override
+    public void testAddColumnWithComment()
+    {
+        throw new SkipException("Clickhouse connector does not support specifying a comment when adding a column");
```

Comment:
This should be handled by declaring that it doesn't have SUPPORTS_ADD_COLUMN_WITH_COMMENT behavior, and checked that the connector indeed rejects column with a comment.

Reply:
Actually the clickhouse connector test supports adding columns only for `MergeTree` engine tables:

https://github.com/trinodb/trino/blob/70ad1f98924561191fb49395cc3398b894f58eea/plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java#L144-L145

However, the column comment is ignored at the moment. This is the reason why the implementation of the test for the Clickhouse connector simply skips testing the functionality.